templates/scripts/import-cf-keypair.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Inputs:
+# keypair ID
+# keypair private key
```

Comment:
The private key you get after creating it in AWS is a file in PEM format, it's probably most convenient to just have them provide a file name as the second arg and then down below we `cat` it and pipe to `tr`?

Reply:
oops, haha, you already did that. I was just looking at the commits in order. 👍